src/logging.ts

```diff
@@ -177,17 +175,15 @@ export function configure(opts: LoggerConfig) {
     // reconfigure any existing loggers. They may have been lazily loaded
     // with the default config, which is now being overwritten by this
     // configure() call.
-    Object.keys(loggers).forEach(function(loggerName) {
-        const existingLogger = loggers[loggerName];
-        // remove each individual transport
+    for (const [, existingLogger] of loggers) {
```

Comment:
```suggestion
    for (const existingLogger of loggers.values()) {
```

Reply:
I've used the above syntax in a lot of places.
Because this is totally bike shedding territory (in my opinion), I wonder if there's an ESLint rule to at least enforce consistency.